packages/imperative/src/config/src/ProfileInfo.ts

```diff
@@ -1388,6 +1388,65 @@ export class ProfileInfo {
         return finalSchema;
     }
 
+    // _______________________________________________________________________
+    /**
+     * List of secure properties with more details, like value, location, and type
+     *
+     * @param opts The user and global flags that specify one of the four
+     *             config files (aka layers).
+     * @returns Array of secure property details
+     */
+    public secureFieldsWithDetails(opts?: { user: boolean; global: boolean }): IProfArgAttrs[] {
```

Comment:
Curious to get your thoughts on making `user` and `global` optional? We can fallback to `false` if the property is not provided, but this would allow a developer to call `secureFieldsWithDetails` without having to specify both `user` and `global`:

```ts
profileInfo.secureFieldsWithDetails({ user: true }); // project user config
// or...
profileInfo.secureFieldsWithDetails({ global: true }); // global, non-user config
```

Reply:
Captured it in:
- https://github.com/zowe/zowe-cli/issues/2247